ekncontent/tests/ekncontent/testAudioObjectModel.js

```diff
@@ -0,0 +1,44 @@
+const Eknc = imports.gi.EosKnowledgeContent;
+
+const InstanceOfMatcher = imports.tests.InstanceOfMatcher;
+
+const MOCK_AUDIO_DATA = {
+    '@id': 'ekn://BagRaiders/ShootingStars',
```

Comment:
Better to use a valid EKN ID here in case later we add validation somewhere in the pipeline.